packages/components/select/__tests__/select.spec.ts

```diff
@@ -882,6 +883,56 @@ describe('Select', () => {
     mockInputWidth.mockRestore()
   })
 
+  test('multiple select with collapseTagsTooltip', async () => {
+    wrapper = _mount(
+      `
+      <el-select v-model="selectedList" multiple collapseTags collapse-tags-tooltip placeholder="请选择">
+        <el-option v-for="item in options" :key="item.value" :label="item.label" :value="item.value">
+        </el-option>
+      </el-select>
+    `,
+      () => ({
+        options: [
+          {
+            value: '选项1',
+            label: '黄金糕',
+          },
+          {
+            value: '选项2',
+            label: '双皮奶',
+          },
+          {
+            value: '选项3',
+            label: '蚵仔煎',
+          },
+          {
+            value: '选项4',
+            label: '龙须面',
+          },
+          {
+            value: '选项5',
+            label: '北京烤鸭',
+          },
+        ],
+        selectedList: [],
+      })
+    )
+    await wrapper.find('.select-trigger').trigger('click')
+    const options = getOptions()
+
+    options[0].click()
+    await nextTick()
+    options[1].click()
+    await nextTick()
+    options[2].click()
+    await nextTick()
```

Comment:
There should only be one `await nextTick()`

Reply:
ok

Reply:
There seems to be a problem with this modification, the test fails...

Reply:
，，，it seems  `await nextTick()`  is required